ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -63,13 +65,14 @@ Serve Model
         Set Folder Path    ${model_path}
     ELSE
         # Existing connection radio is selected by default; for now blindly click on new connection radio
-        Click Element    //input[@id="new-data-connection-radio"]
+        SeleniumLibrary.Click Element    //input[@id="new-data-connection-radio"]
         # Select Radio Button    group_name=radiogroup    value=new-data-connection-radio
         Set Up New Data Connection    dc_name=${data_connection_name}
         Set Folder Path    ${model_path}
     END
-    Click Button    Deploy
-    Wait Until Page Does Not Contain    xpath://h1[.="Deploy model"]
+    Sleep    5s
```

Comment:
Any chance there is a possible to wait for a particular element on the page to show up instead of wait? :slightly_smiling_face:

Reply:
ops, this was one tentative to fix it, but it's not needed anymore :) thanks for catching it